reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    mem_present = 1                 # flag to include membrane (0 = no membrane, 1 = membrane)
</pre><p>Likewise, I think this should be a Boolean. Since this is meant to be an example, can you spell out the variable names (<code>mem</code> -&gt; <code>membrane</code> ) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Modified this variable to a Boolean.</p>